src/main/resources/wrong-secrets-configuration.yaml

```
@@ -827,3 +827,16 @@ configurations:
       category: *secrets
       ctf:
         enabled: true
+
+    - name: Challenge 53
+      short-name: "challenge-53"
+      sources:
+        - class-name: "org.owasp.wrongsecrets.challenges.docker.Challenge53"
+          explanation: "explanations/challenge53.adoc"
+          hint: "explanations/challenge53_hint.adoc"
+          reason: "explanations/challenge53_reason.adoc"
+          environments: *all_envs
+      difficulty: *normal
+      category: *secrets
+      ctf:
+        enabled: true
```

```suggestion
        enabled: false
```
we don't have it onboarded ot ctf party yet